packages/cli/src/commands/server/deploy.ts

```
@@ -14,6 +14,8 @@ import { getToken, getCurrentOrgId } from '../auth/credentials.js';
 import { preflightBuildOutput, printPreflightIssues } from '../deploy-preflight.js';
 import { getProjectConfigToSave, loadProjectConfig, saveProjectConfig } from '../studio/project-config.js';
 import { fetchServerProjects, createServerProject, uploadServerDeploy, pollServerDeploy } from './platform-api.js';
+import { MASTRA_PLATFORM_API_URL } from '../auth/client.js';
```

ESLint

[error] `../auth/client.js` import should occur before import of `../auth/credentials.js`

(import/order)